[ADD] global style 생성 및 로그인 페이지 스타일링 적용

pje1740 opened the pull request:
[comment]: <> (PR 태그를 명시해주세요. [ADD] / [UPDATE] / [BUG])

### 요약
범용적으로 사용되는 스타일은 style/globalStyle.js에서 선언한 뒤 App.js에서 불러오고 있습니다.
로그인 페이지와 관련된 스타일링은 LoginPage.js에서 모두 적용하고 있습니다.

### 핵심 로직 및 내용
아이콘은 Material icon 모듈을 설치하여 사용하고 있습니다. 
이후에 필요하신 아이콘이 있다면 그걸 불러와서 사용하면 됩니다.

### 기타 참고 사항
스타일링이 많이... 더럽습니다...

pje1740 committed:
[CHORE] install @material-ui/icons

[why]
We will use material icons for icon sections

[what]
mateiral-ui/icons module

[etc]

pje1740 committed:
[CHORE] install material ui core module

[why]
To use material icons

[what]
mateiral ui core module.

[etc]

pje1740 committed:
[FEAT] apply global styles

[why]
These styles are globally required

[what]
Font - Open sans
border box
body margin & padding
h3

[etc]

pje1740 committed:
[FEAT] apply style to LoginPage

[why]
styling

[what]
LoginPage.js

[etc]

JeesooHaa commented on Frontend/style/globalStyles.js:
```diff
@@ -0,0 +1,22 @@
+import { createGlobalStyle } from 'styled-components';
```
global style은 이렇게 하는거군요 👍

thesulks commented on Frontend/Views/App.js:
```diff
@@ -18,6 +20,7 @@ const App = () => {
 
   return (
     <>
+      <GlobalStyle />
```
👍❗️

pje1740 committed:
[FEAT] add :root in globalStyle for global variables.

[why]
Some of the values or styles need to be shared across pages.

[what]
globalStyles.js - :root - colors

[etc]

pje1740 committed:
[FEAT] create global color schemes

[why]
These colors will be shared and resued across pages

[what]
login-gray
border-gray
background-gray
button-green
header-black
list-gray
open-green
closed-red
owner-blue
tab-blue

[etc]
